src/pages/settings/Wallet/ExpensifyCardPage.js

```diff
@@ -73,13 +81,29 @@ function ExpensifyCardPage({
                             interactive={false}
                             titleStyle={styles.newKansasLarge}
                         />
-                        {!_.isEmpty(physicalCard) && (
-                            <MenuItemWithTopDescription
-                                description={translate('cardPage.virtualCardNumber')}
-                                title={CardUtils.maskCard(virtualCard.lastFourPAN)}
-                                interactive={false}
-                                titleStyle={styles.walletCardNumber}
+                        {shouldShowCardDetails ? (
+                            <CardDetails
+                                pan="1234123412341234"
+                                expiration="11/02/2024"
+                                cvv="321"
                             />
+                        ) : (
+                            !_.isEmpty(physicalCard) && (
```

Comment:
Should wrap the whole block with it? (in this shouldShowCardDetails will be inside block with isEmpty check)

If we will miss the virtual card - we should hide everything related to it  

p. s.
Looks like it is typo from the merged commit we should have `virtualCard` inside this statement

Reply:
Done, let me know what you think